defaults/main.yml

```diff
@@ -14,7 +14,7 @@ docker_dependencies: "{{ default_docker_dependencies }}"
 
 # You can set any interface, that is listened by docker engine.
 # e.g. docker_swarm_interface: "eth1"
-docker_swarm_interface: "{{ ansible_default_ipv4['interface'] }}"
+docker_swarm_interface: "{{ docker_swarm_interface | ansible_default_ipv4['interface'] }}"
```

Comment:
This can not work. Everything in `{{` and `}}` is handled like in Jinja. And in jinja you have `{{ variablename | filter }}`. That's why the tests fail.